test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2939,10 +2961,12 @@ class test_ucp_sockaddr_protocols_diff_net_devices :
      */
     void create_entities_and_connect()
     {
+        UCS_TEST_MESSAGE << "client";
```

Comment:
still needed?

Reply:
optional, IMHO, this looks more informative:
```
[ RUN      ] rcx/test_ucp_sockaddr_protocols_diff_net_devices.restricted_net_devices/0 <rc_x,cuda_copy,rocm_copy/restrict_client>
[     INFO ] client
[     INFO ] NET_DEVICES=ibs1,mlx5_1:1 (1.1.80.1)
[     INFO ] server
[     INFO ] NET_DEVICES=all
[     INFO ] server listening on 1.1.80.1:47561
[     INFO ] ignoring error Connection reset by remote peer on endpoint 0x7f90c705a000
[       OK ] rcx/test_ucp_sockaddr_protocols_diff_net_devices.restricted_net_devices/0 (549 ms)
```

Reply:
maybe print in same line:
```
[ RUN      ] rcx/test_ucp_sockaddr_protocols_diff_net_devices.restricted_net_devices/0 <rc_x,cuda_copy,rocm_copy/restrict_client>
[     INFO ] client using NET_DEVICES=ibs1,mlx5_1:1 (1.1.80.1)
[     INFO ] server using NET_DEVICES=all
[     INFO ] server listening on 1.1.80.1:47561
[     INFO ] ignoring error Connection reset by remote peer on endpoint 0x7f90c705a000
[       OK ] rcx/test_ucp_sockaddr_protocols_diff_net_devices.restricted_net_devices/0 (549 ms)
```

Reply:
done